### DISCUSSION TITLE
log 增加过滤功能

### DISCUSSION THREAD

**Discussion by Jiematcha** [v2fly/v2ray-core#1584] [Ideas]
不显示或只显示特定域名或协议或接口，便于排错

**Comment by afxMetoo** [comment]
你用的是linux系统吗，如果是的话可以用linux自带的工具来过滤掉，具体为: tail -f /var/log/v2ray/error.log | grep -a "你的关键词1" | grep -a "关键词2" 可以无限筛选。
关键命令: tail 、grep . 相当好用

**Reply by swoiow** [reply]
可以使用grep -v 排除
